config/initializers/active_job_select_foodcoop.rb

```diff
@@ -0,0 +1,22 @@
+module FoodsoftActiveJobArguments
+  def self.included(base) # :nodoc:
+    base.class_eval do
+      alias_method :orig_deserialize, :deserialize
+      alias_method :orig_serialize, :serialize
+
+      def serialize(arguments)
+        ret = orig_serialize(arguments)
+        ret.unshift FoodsoftConfig.scope
+      end
+
+      def deserialize(arguments)
+        FoodsoftConfig.select_multifoodcoop arguments.shift
+        orig_deserialize(arguments)
+      end
+    end
+  end
+end
+
+ActiveSupport.on_load(:after_initialize) do
+  ActiveJob::Arguments.include FoodsoftActiveJobArguments
+end
```

Comment:
Cool. I would include this manually in each job, if possible. This allows people to write a Foodsoft plugin that does _not_ use a foodcoop scope (e.g. an overview of all instances).

Reply:
nope, since we need it for the internal rails jobs (e.g. for active storage)

Reply:
Good point. Then we'll need find a solution later if such a plugin appears.